tests/analyzer/sasl-encrypted.zeek

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2021 by the Zeek Project. See LICENSE for details.
+
+# @TEST-EXEC: zeek -C -r ${TRACES}/ldap-krb5-sign-seal-01.pcap %INPUT
```

Comment:
Could you add information where the PCAP came from to `tests/traces/README`? This can be very short if you created it yourself.

Reply:
@bbannier done: I added a note to the commit message and also in a comment in `tests/analyzer/sasl-encrypted.zeek`

Reply:
Thanks, I'll move this to `tests/traces/README` while committing.